javascript-functions/main.js

```diff
@@ -0,0 +1,37 @@
+function convertMinutesToSeconds(minutes) {
+  var seconds = minutes * 60;
+  return seconds;
+}
+
+console.log('convertMinutesToSeconds:', convertMinutesToSeconds(5));
+
+function greet(name) {
+  var greeting = 'Hey ' + name + '!';
+  return greeting;
+}
+
+console.log('greet: ', greet('Hien'));
+
+function getArea(width, height) {
+  var area = width * height;
+  return area;
+}
+
+console.log('getArea: ', getArea(17, 42));
+
+function getFirstName(person) {
+  var name = {
+    firstName: person.firstName,
+    lastName: person.lastName
+  };
```

Comment:
Instead of doing this extra step of making a new object, you should just return `person.firstName`